.github/workflows/test.yml

```diff
@@ -1,5 +1,9 @@
 name: Test
 on: [pull_request]
+env:
+  # 7 GiB by default on GitHub, setting to 6 GiB
+  # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
+  NODE_OPTIONS: --max-old-space-size=6144
```

Comment:
Any reason for decreasing this from 7?

Reply:
Weirdly this allows the runner to access more of its memory, I don't know why, and it turns out there's a Vite bug of all things that occasionally causes increased memory usage to the point that the SB build kept tripping up.

Reply:
lol, cool